test/convert_test.go

```diff
@@ -63,13 +64,19 @@ func TestConvert(t *testing.T) {
 			t.Fatalf("unmarshaling: %v", err)
 		}
 
-		assetsByType := make(map[string][]google.Asset)
+		assetsByType := make(map[string]google.Asset)
 		for _, a := range assets {
-			assetsByType[a.Type] = append(assetsByType[a.Type], a)
+			t := a.Type
+			// NOTE: Since IAM policy assets will have the same type, we'll add a
+			// suffix to map it to the right JSON fixture
+			if a.IAMPolicy != nil {
+				fmt.Println("POLICY")
```

Comment:
This should probably either be dropped or use t.Log/Logf.

Reply:
Good catch